_drafts/release-osiam-3.0.md

```diff
@@ -3,81 +3,124 @@ title:  "Release: OSIAM 3.0"
-
-TODO: Write me
+It follows an overview of the most important changes.
+See the full changelog of [OSIAM 3.0](https://github.com/osiam/osiam/blob/3.0/CHANGELOG.md) for more information.
 
 ## Deployment and Configuration
 
-TODO: Write me
-
-### YAML configuration
-
-TODO: Write me
-
-### Merge of the Servers
+**NOTE:** This part of OSIAM changed heavily, making it incompatible with any previous version.
+For a complete overview of the necessary steps to update, see the [migration notes](https://github.com/osiam/osiam/blob/master/docs/migration.md#from-25-to-30).
+It's also recommended to read the complete [installation guide](https://github.com/osiam/osiam/blob/master/docs/detailed-reference-installation.md)
+once again to fully understand the new features and changes.
+
+Until now, OSIAM has been distributed as a bunch of distinct services.
+There were the auth-server and resource-server, that formed the required core of OSIAM.
+And there were the add-ons, that extended OSIAM with an administration, a registration process and much more.
+With OSIAM 3.0, this is not the case anymore.
+We've decided to make OSIAM a single application again.
+You can get more information about our decision in the corresponding [GitHub issue](https://github.com/osiam/osiam/issues/43).
+As this is a complex undertaking, we started it with just merging the core services,
+i.e the auth-server and resource-server.
+So from now on, the core functions of OSIAM are distributed as a single package containing only 1 service.
+
+There are also no distribution archives anymore.
+The only artifact that we distribute is the OSIAM `.war` file.
+This file already contains everything OSIAM needs, e.g. templates and web assets.
+These files will be automatically extracted in a new "home directory", when OSIAM starts.
+But you can still extract them manually if you like.
+The home directory can be placed anywhere you like, but defaults to `~/.osiam`.
+It is the place where everything OSIAM needs at runtime lives now.
+This also includes the configuration file, making it a replacement for `/etc/osiam` in a default 2.x installation.
+
+Speaking of configuration file: We changed the supported format for it from a Java properties file to YAML.
+This means you have to rewrite your current configuration files, making it a single YAML file.
+Fortunately, it's very easy to do so.
+See the [migration notes](https://github.com/osiam/osiam/blob/master/docs/migration.md#create-a-new-configuration-file)
+for more details.
+
+Another great feature of OSIAM 3.0 is, that you don't need a Tomcat server anymore.
+While deploying OSIAM to Tomcat is still possible, you can also run it as a standalone application now.
+Just start OSIAM with `java -jar osiam.war`.
+But that's not all.
+The `.war` file is not only an executable Java web archive, it's actually a fully executable file.
+So, if you set the executable permission on it, you can start it like any other application on the command line:
```

Comment:
If executable permissions are set on it, it can be started like any other application on the command line: